pallets/capacity/src/tests/mock.rs

```diff
@@ -235,7 +235,7 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
 	let mut ext = sp_io::TestExternalities::new(t);
 	ext.execute_with(|| {
 		System::set_block_number(1);
-		set_era_and_reward_pool_at_block(1, 0, 0);
+		set_era_and_reward_pool(1, 1, 0);
```

Comment:
it made more sense to start at era 1 and have that start at block 1 instead of block 0, because block number is also 1.